app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,26 @@
+class CommentsController < ApplicationController
+  def index
+    @post = Post.find(params[:post_id])
+    @user = User.find(params[:user_id])
+  end
+
+  def new
+    @comment = Comment.new
+    respond_to do |format|
+      format.html { render :new, locals: { comment: @comment } }
+    end
+  end
+
+  def create
+    @user = User.find(params[:user_id])
+    @post = Post.find(params[:post_id])
+    @comment = Comment.new(user: @user, post: @post, text: params[:comment][:text])
+    if @comment.save
+      redirect_to comments_path
+    else
+      render 'new', locals: { comment: @comment }
+    end
+  end
+
+  def show; end
```

Comment:
- [ ] Let's add strong parameters in all your controllers. You may check this [lesson](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides.) again to know why we need it.

it will look like this:
![Screenshot from 2022-02-26 17-26-27](https://user-images.githubusercontent.com/65860678/155848773-c15d0832-7b76-4f83-a9b7-5b605aabc1e2.png)

- [ ] You should also consider removing lines that are not related to the current project.
![Screenshot from 2022-02-26 17-30-50](https://user-images.githubusercontent.com/65860678/155848896-e229a318-1de8-4db0-a25e-5c867e97082d.png)